lms/templates/help_modal.html

```diff
@@ -19,196 +19,231 @@
 </div>
 
 <section id="help-modal" class="modal" aria-hidden="true" role="dialog" aria-label="${_("{platform_name} Help").format(platform_name=static.get_platform_name())}">
-  <div class="inner-wrapper" id="help_wrapper">
+  <div class="inner-wrapper">
     ## TODO: find a way to refactor this
-    <button class="close-modal "tabindex="0">
+    <button class="close-modal" tabindex="0">
       <span class="icon fa fa-remove" aria-hidden="true"></span>
```

Comment:
Anyway we can fix the contrast on this button? It's white on light grey when not focused.